src/Jackett.Common/Definitions/siambit.yml

```diff
@@ -7,7 +7,7 @@ type: private
 encoding: tis-620
 requestDelay: 2
 links:
-  - https://siambit.io/
```

Comment:
please put `  - https://siambit.io/` under the legacylinks so that users indexer configs can automatically be updated to point to the new domain.

Reply:
> please put ` - https://siambit.io/` under the legacylinks so that users indexer configs can automatically be updated to point to the new domain.

I have fixed it.

Reply:
Thanks. I meant added to the legacy without replacing the old one, but anyway I've amended it.